### ISSUE TITLE
Animated instance icons

### ISSUE THREAD

**Issue by MrPotatoisgod** [PrismLauncher/PrismLauncher#2961]
### Role

I like to use custom icons for instances

### Suggestion

I wish for the ability when selecting apngs and gifs for an instance's icon, for them to be animated

### Benefit

So that apngs and gifs can be used to their full potential

### This suggestion is unique

- [X] I have searched the issue tracker and did not find an issue describing my suggestion, especially not one that has been rejected.

### You may use the editor below to elaborate further.

Basically just animated instance icon support

**Comment by Trial97** [comment]
Duplicate of: https://github.com/PrismLauncher/PrismLauncher/issues/2742
